plugins/jvm-example/src/main/java/org/elasticsearch/plugin/example/JvmExamplePlugin.java

```diff
@@ -111,5 +117,16 @@ public Settings additionalSettings() {
     public void onModule(RepositoriesModule repositoriesModule) {
     }
 
-
+    /**
+     * Module decalaring some example configuration and a _cat action that uses
+     * it.
+     */
+    public static class ConfiguredExampleModule extends AbstractModule {
+        @Override
+        protected void configure() {
+          bind(ExamplePluginConfiguration.class).asEagerSingleton();
+          Multibinder<AbstractCatAction> catActionMultibinder = Multibinder.newSetBinder(binder(), AbstractCatAction.class);
```

Comment:
> think it is more confusing to have some things bound using a binder, and others with registration/settings.

Yeah - but we're based on Guice. I think that hiding Guice under the covers is just asking for trouble.

Preventing multiple Multibinders from working would just handcuff plugin authors.